<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Check_out</class>
 <widget class="QWidget" name="Check_out">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Room_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;等线 Light&quot;;</string>
   </property>
   <property name="text">
    <string>请选择房间号：</string>
   </property>
  </widget>
  <widget class="QWidget" name="Check_widget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <widget class="QLabel" name="Cost_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;等线 Light&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Check_Btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pic/check_out.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Ret_Btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pic/Ret.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;等线 Light&quot;;</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;等线 Light&quot;;</string>
    </property>
    <property name="text">
     <string>办理离店</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;等线 Light&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>101</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>102</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>103</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>104</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>105</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="SureBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/Login.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/Hotel.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>Room_label</zorder>
  <zorder>comboBox</zorder>
  <zorder>SureBtn</zorder>
  <zorder>Check_widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
